ionos_prd/namespaces/chatbot-backend.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
```

Comment:
Namespaces for PRD are generically defined in folder ionos_common/namespaces
Please add resources there and rebase the changes

Reply:
Please remove that one. The comment above goes for cs-identity namespace too

Reply:
also , please rebase the changes and squash the commits